packages/ai/ai/src/McpServer.ts

```diff
@@ -632,7 +632,7 @@ export const registerToolkit: <Tools extends AiTool.Any>(toolkit: AiToolkit.AiTo
     yield* registry.addTool({
       tool: mcpTool,
       handle(payload) {
-        return built.handle(tool.name as any, payload).pipe(
+        return built.handle("", tool.name as any, payload).pipe(
```

Comment:
In particular, this feels bad to me.

Reply:
Wouldn’t the rpc request id be fine here?